tests/wrapper/test_simulation_objects.py

```diff
@@ -30,6 +30,36 @@ def test_update_state(self, traffic_update):
         assert signal.state == Signal.State.GO
 
 
+class TestTrack:
+    """Tests for the track component"""
+
+    @pytest.fixture
+    def track(self):
+        return Track("track")
+
+    @pytest.fixture
+    def speed_update(self, monkeypatch):
+        def set_max_speed(identifier: str, speed: float) -> None:
+            assert identifier == "track"
+            assert speed == 100
+
+        monkeypatch.setattr(edge, "setMaxSpeed", set_max_speed)
+
+    def test_update_speed(self, track, speed_update):
+        # pylint: disable=unused-argument
+        track.max_speed = 100
```

Comment:
Shouldn't the setter method be used here?

Reply:
It should use the setter automatically, thanks to [this](https://github.com/BP2022-AP1/bp2022-ap1/pull/210/files/e20e8d4091060d5b77701fab7fcb912f1d537129#diff-32b59e5349657358f01749afb728eedcd06c4ca2ca854f0b475cc29a31aa8ef5R119) annotation

Reply:
Ah cool, I didn't know that. 🤯